hack/e2e-setup-kind-cluster.sh

```diff
@@ -15,8 +15,8 @@ done
 
 HERE="$(dirname "$(readlink --canonicalize ${BASH_SOURCE[0]})")"
 ROOT="$(readlink --canonicalize "$HERE/..")"
-MULTUS_DAEMONSET_URL="https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/deployments/multus-daemonset.yml"
-CNIS_DAEMONSET_URL="https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/e2e/cni-install.yml"
+MULTUS_DAEMONSET_URL="https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/v3.9.1/deployments/multus-daemonset.yml"
+CNIS_DAEMONSET_URL="https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/v3.9.1/e2e/cni-install.yml"
```

Comment:
What error do you currently see in CI ? 

I worry a bit about this change, because it implies we have to track multus releases ... Something which I would rather not have to do.

@e0ne

Reply:
@e0ne I only now realized you're pushing this directly to `release-v0.5` branch; wouldn't it make more sense to merge it to master/main branch, and then backport it into the release branch ?...

If we just do this, the main branch would still have a deprecated attribute.

Reply:
@e0ne would you please update change this PR so it merges in the master branch ? We can backport to v0.5.0 afterwards.